be/src/storage/rowset/column_iterator.h

```diff
@@ -52,7 +54,9 @@ class ColumnReader;
 class RandomAccessFile;
 
 struct ColumnIteratorOptions {
-    RandomAccessFile* read_file = nullptr;
+    //RandomAccessFile* read_file = nullptr;
+    io::SeekableInputStream* read_file = nullptr;
```

Comment:
can we just use `RandomAccessFile::stream()` to get the `io::SeekableInputStream`?
```suggestion
    RandomAccessFile* read_file = nullptr;
```